src/__tests__/toPropertyPath.spec.ts

```diff
@@ -7,4 +7,5 @@ test('toPropertyPath', () => {
   expect(toPropertyPath('/paths/~1~1~1~1pets/post')).toEqual('paths.////pets.post');
   expect(toPropertyPath('/paths/foo~0users')).toEqual('paths.foo~users');
   expect(toPropertyPath('#/definitions/Error')).toEqual('definitions.Error');
+  expect(toPropertyPath('/a/..dotted/path')).toEqual('a.["..dotted"].path');
```

Comment:
Out of curiosity can a path be like: `'/a/.."dotted"/path'`?

Reply:
Good catch! Fixed